reference/events.rst

```diff
@@ -215,10 +215,12 @@ and set a new ``Exception`` object, or do nothing::
     response won't work. If you want to overwrite the status code (which you
     should not without a good reason), set the ``X-Status-Code`` header::
 
-        $response = Response(
+        $response = new Response(
             'Error',
-            404 // ignored,
-            array('X-Status-Code' => 200)
+            404 // this status code will be ignored,
```

Comment:
the comma should go after the `404` number to avoid syntax errors (previously was also wrong of course)